### PR TITLE
Add and enable mraa GPIO tests

### PR DESCRIPTION
**Pull request by skuusela**
These tests will set a GPIO pin on and off and check the output with BeagleBone that it works.

### CODE REVIEW

**Comment by skuusela** [comment]
test this please

**Comment by skuusela** [comment]
@mythi This is kinda the same problem as with adding Bluetooth test but this test has to be hardware specific. I also made alternative way of doing this without using the mask files: https://github.com/skuusela/intel-iot-refkit/commit/59f80e77aafc491ead495de7f1dbacb9e7b07211.

**Comment by mythi** [comment]
How reliable is mraa's capability to detect the board? See ```mraa-gpio version```

**Comment by skuusela** [comment]
@mythi On Joule it gives: "Version v1.5.1 on Intel GT Tuchuck" and on Minnowboard Turbot: "Version v1.5.1 on MinnowBoard MAX"

**Comment by mythi** [comment]
Oh, I did not actually check https://github.com/skuusela/intel-iot-refkit/commit/59f80e77aafc491ead495de7f1dbacb9e7b07211 before my guestion. I'd prefer that option because it also tests mraa's capability to detect the board at the same time.

**Comment by mythi** [comment]
test this please

File: meta-iotqa/lib/oeqa/runtime/sanity/mraa_gpio.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        else:
+            raise unittest.SkipTest(output)
+
+    def test_gpio(self):
+        '''
+        Test a GPIO pin on and off and check the pin output with
+        BeagleBone
+        '''
+        def check_gpio_output():
+            cmd = "cat /sys/class/gpio/gpio20/value".split()
```
Comment (mythi) [inline meta-iotqa/lib/oeqa/runtime/sanity/mraa_gpio.py]: can you do this directly without exporting the GPIO first?

Comment (skuusela) [inline meta-iotqa/lib/oeqa/runtime/sanity/mraa_gpio.py]: No the GPIO pin has to be exported, but it's already exported on the Beaglebones. I added few lines to a boot script on the Beaglebone so it's always exported and ready after booting.

**Pull request merged**